src/Details.jl

```diff
@@ -0,0 +1,246 @@
+### A Pluto.jl notebook ###
+# v0.19.38
+
+using Markdown
+using InteractiveUtils
+
+# This Pluto notebook uses @bind for interactivity. When running this notebook outside of Pluto, the following 'mock version' of @bind gives bound variables a default value (instead of an error).
+macro bind(def, element)
+    quote
+        local iv = try Base.loaded_modules[Base.PkgId(Base.UUID("6e696c72-6542-2067-7265-42206c756150"), "AbstractPlutoDingetjes")].Bonds.initial_value catch; b -> missing; end
+        local el = $(esc(element))
+        global $(esc(def)) = Core.applicable(Base.get, el) ? Base.get(el) : iv(el)
+        el
+    end
+end
+
+# ╔═╡ 81f5b495-76c4-4c54-93ab-b49c5ecb810a
+# ╠═╡ skip_as_script = true
+#=╠═╡
+begin
+	import Pkg
+	Pkg.activate(Base.current_project(@__DIR__))
+	Pkg.instantiate()
+	md"**Project env active**"
+end
+  ╠═╡ =#
+
+# ╔═╡ b7b18a54-afd7-4467-83ed-cc4f07c321fb
+using HypertextLiteral
+
+# ╔═╡ b3732e34-d331-4dd2-b4fb-11b2f397d7c1
+# ╠═╡ skip_as_script = true
+#=╠═╡
+const testslider = html"<input>"
+  ╠═╡ =#
+
+# ╔═╡ 13e81634-3b72-4b1d-a89b-36d184698d21
+const details_css = @htl("""
+<style type="text/css">
+pluto-output details {
+	border: 1px solid var(--rule-color);
+	border-radius: 4px;
+	padding: 0.5em 0.5em 0;
+	margin-block-start: 0;
+	margin-block-end: var(--pluto-cell-spacing);
+}
+
+pluto-output details:first-child {
+	margin-block-start: 0;
+}
+
+pluto-output details:last-child {
+	margin-block-end: 0;
+}
+
+pluto-output details summary {
+	cursor: pointer;
+	font-weight: bold;
+	margin: -0.5em -0.5em 0;
+	padding: 0.5em;
+	font-family: var(--system-ui-font-stack);
+}
+
+pluto-output details[open] {
+	padding: 0.5em;
+}
+
+pluto-output details[open] summary {
+	border-bottom: 1px solid var(--rule-color);
+	margin-bottom: 0.5em;
+}
+</style>
+""")
+
+# ╔═╡ df840588-23bd-4b03-b5ab-ef273052d198
+const Iterable = Union{AbstractVector, Tuple, Base.Generator}
+
+# ╔═╡ 46521e2b-ea06-491a-9842-13dff7dc8299
+begin
+	const embed_detail = isdefined(Main, :PlutoRunner) && isdefined(Main.PlutoRunner, :embed_display) ? Main.PlutoRunner.embed_display : identity
```

Comment:
FYI this can later be replaced with https://github.com/JuliaPluto/AbstractPlutoDingetjes.jl/pull/9